---
title: "Investigation of the usefulness of liver-specific deconvolution method by establishing a liver benchmark dataset"
collection: publications
permalink: /publication/2024-01-05-paper-title-number-4
excerpt: ''
date: 2024-01-05
venue: 'NAR Genomics and Bioinformatics'
paperdoi: 'doi.org/10.1093/nargab/lqad111'
paperurl: 'https://academic.oup.com/nargab/article/6/1/lqad111/7511730'
citation: 'Azuma, Iori, et al. "Investigation of the usefulness of liver-specific deconvolution method by establishing a liver benchmark dataset." NAR Genomics and Bioinformatics (2024): 2024-03.'
---
Background Immune responses in the liver are related to the development and progression of liver failure, and precise prediction of their behavior is important. Deconvolution is a methodology for estimating the immune cell proportions from the transcriptome, and it is mainly applied to blood-derived samples and tumor tissues. However, the influence of tissue-specific modeling on the estimation results has rarely been investigated. In this study, we constructed a system to evaluate the performance of the deconvolution method on liver transcriptome data.

Results We prepared seven mouse liver injury models using small-molecule compounds with known hepatotoxicity and established a dataset with corresponding liver bulk RNA-Seq and immune cell proportions, covering various immune responses. RNA-Seq expression for nine leukocyte subsets and four liver-associated cell types were obtained from the Gene Expression Omnibus (GEO) to provide a reference covering liver component cells. Here, we found that the combination of reference cell sets affects the estimation results of reference-based deconvolution methods. We established a liver tissue-specific deconvolution by optimizing the reference cell set for each cell to be estimated. We applied this model to independent datasets and showed that liver-specific modeling focusing on reference cell sets is highly extrapolatable.

Conclusions We provide an approach of liver-specific modeling when applying reference-based deconvolution to bulk RNA-Seq data and show its importance. It is expected to enable sophisticated estimation from rich tissue data accumulated in public databases and to obtain information on aggregated immune cell trafficking.

[Download paper here](https://academic.oup.com/nargab/article/6/1/lqad111/7511730)
