---
title: "GLDADec: marker-gene guided LDA modelling for bulk gene expression deconvolution"
collection: publications
permalink: /publication/2024-01-09-paper-title-number-5
excerpt: ''
date: 2024-01-09
venue: 'bioRxiv'
paperdoi: 'doi.org/10.1101/2024.01.08.574749'
paperurl: 'https://www.biorxiv.org/content/10.1101/2024.01.08.574749v1'
citation: 'Azuma, Iori, et al. "GLDADec: marker-gene guided LDA modelling for bulk gene expression deconvolution." bioRxiv (2024)'
---
Inferring cell type proportions from bulk transcriptome data is crucial in immunology and oncology. Here, we introduce GLDADec (Guided LDA Deconvolution), a bulk deconvolution method that guides topics using cell type-specific marker gene names to estimate topic distributions for each sample. Through benchmarking using blood-derived datasets, we demonstrate its high estimation performance and robustness. Moreover, we apply GLDADec to heterogeneous tissue bulk data and perform comprehensive cell type analysis in a data-driven manner. We show that GLDADec outperforms existing methods in estimation performance and evaluate its biological interpretability by examining enrichment of biological processes for topics. Finally, we apply GLDADec to TCGA tumor samples, enabling subtype stratification and survival analysis based on estimated cell type proportions, thus proving its practical utility in clinical settings. This approach, utilizing marker gene names as partial prior information, can be applied to various scenarios for bulk data deconvolution. GLDADec is an available as an open-source Python package at https://github.com/mizuno-group/GLDADec.

[Download paper here](https://www.biorxiv.org/content/10.1101/2024.01.08.574749v1)
